website/versioned_docs/version-0.19.0/searchbar.md

```diff
@@ -227,5 +227,5 @@ You can then access SearchBar methods like so:
 ```
 this.search.focus();
 this.search.blur();
-this.search.clearText();
+this.search.clear();
```

Comment:
Change this back to `clearText`, that was the name of the method in 0.19.0 https://github.com/react-native-training/react-native-elements/blob/v0.19.1/src/input/Search.js#L45